src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -376,6 +376,7 @@ export default {
     loadAnnualReport (filing: any) {
       const header = filing?.header
       const annualReport = filing?.annualReport
+      const documents = filing?.documents
 
       if (header && annualReport) {
```

Comment:
Should you ensure 'documents' is truthy here?

Also in other methods below.

Reply:
I do ```documents: documents || [] as Array<any>``` when assigning it to the item object, is that okay?

Reply:
You do that so the template for loop doesn't fail if there are no documents.

I'm actually wondering if 'documents' are required in all filings in the history list? I suspect they are (except for paper only).

Reply:
Currently the api will return an empty array if the logic determines there are no relevant documents to return.

Reply:
Cool. So I suggest doing the same thing as with 'comments':
- don't bother ensuring it's truthy on line 380
- move the assignment on line 379 to line 398 (the filings obj) and keep your fallback safety check ([])
- do the same for the other filing types

Reply:
Updated. Thanks!